openwisp_users/base.py

```diff
@@ -1,7 +1,7 @@
 from django.conf import settings
 
 if 'reversion' in settings.INSTALLED_APPS:
-    from reversion.admin import VersionAdmin as BaseModelAdmin
+    from reversion.admin import VersionAdmin as BaseModelAdmin  # pragma: no cover
```

Comment:
the comment should be put one line above as indicated in the [coverage documentation](https://coverage.readthedocs.io/en/coverage-4.3.4/excluding.html):
```python
if 'reversion' in settings.INSTALLED_APPS:  # pragma: no cover
    from reversion.admin import VersionAdmin as BaseModelAdmin
```